packages/boilerplate-generator/boilerplate-generator.js

```diff
@@ -27,12 +25,10 @@ Boilerplate = function (arch, manifest, options) {
 Boilerplate.prototype.toHTML = function (extraData) {
   var self = this;
 
-  if (! self.baseData || ! self.func)
+  if (! self.baseData || ! self.template)
     throw new Error('Boilerplate did not instantiate correctly.');
 
-  return  "<!DOCTYPE html>\n" +
-    Blaze.toHTML(Blaze.With(_.extend(self.baseData, extraData),
-                            self.func));
+  return  "<!DOCTYPE html>\n" + self.template(_.extend(self.baseData, extraData));
```

Comment:
Here's an idea you can ignore for the time being:

I would love to enable more interesting extensions of `self.baseData` here, specifically to support server-side rendering. For example, if this code called some custom callback with `self.baseData`, then the callback could parse the `self.baseData.body` HTML, attach server-rendered fragments to DOM nodes with particular `id`s. Then the client could use `document.getElementById` to find the same nodes and rehydrate the server-generated HTML.

For now, let's focus on feature parity, but I wanted to plant that seed anyway. 🌱